packages/devtools_app/lib/src/performance/flutter/performance_screen.dart

```diff
@@ -60,7 +61,7 @@ class PerformanceBodyState extends State<PerformanceBody> {
         // expand/collapse all by default.
         _data.cpuProfileRoot.expandCascading();
       });
-    });
+    }).catchError((_) {});
```

Comment:
You can fix the error completely by making this change:

Future.delayed(...).then((_) async {
  if (!mounted) return;
  ...
});

Reply:
done.